src/o365/spo/commands/orgassetslibrary/orgassetslibrary-add.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  ContextInfo, ClientSvcResponse, ClientSvcResponseContents
+} from '../../spo';
+import config from '../../../../config';
+import request from '../../../../request';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption,
+  CommandError,
+  CommandValidate
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  libraryUrl: string;
+  thumbnailUrl?: string;
+  cdnType?: string;
+}
+
+class SpoOrgAssetsLibraryAddCommand extends SpoCommand {
+  public get name(): string {
+    return `${commands.ORGASSETSLIBRARY_ADD}`;
+  }
+
+  public get description(): string {
+    return 'Promotes an existing library to become an organization assets library';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.cdnType = args.options.cdnType || 'Private';
+    telemetryProps.thumbnailUrl = typeof args.options.thumbnailUrl !== 'undefined';
+    return telemetryProps;
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    let spoAdminUrl: string = '';
+    const cdnTypeString: string = args.options.cdnType || 'Private';
+    const cdnType: number = cdnTypeString === 'Private' ? 1 : 0;
+    const thumbnailSchema: string = typeof args.options.thumbnailUrl == 'undefined' ? `<Parameter Type="Null" />` : `<Parameter Type="String">${args.options.thumbnailUrl}</Parameter>`;
```

Comment:
Hey @appieschot, I wonder is there a specific reason for not using the strict equal here?

Reply:
Absolutely right, should be strict :-)

Reply:
Would you be able to do that change?